operator/controllers/common/metrics_test.go

```diff
@@ -223,15 +220,9 @@ func TestMetrics_ObserveDeploymentInterval(t *testing.T) {
 			err: nil,
 		},
 		{
-			name:     "previous version - object found but cannot unwrap",
```

Comment:
this test case was removed because this was for testing the error case when the predecessor for a certain item that was retrieved via a GET call to the k8s api could not be handled. However with the retrieval of the predecessor locally from the list of objects we already have, this error case cannot occur anymore